src/ui/layout/section-layout.svelte

```diff
@@ -53,6 +53,18 @@
     }
   }
 
+  @container layout-query-container (width >= 768px) and (width < 1024px) {
+    .layout-content {
```

Comment:
created https://linear.app/revenuecat/issue/WEB-2314/refactor-container-queries-in-purchases-js to track some work that's needed to improve complexity here